src/AvtaleOversikt/AvtalekortMobil.tsx

```diff
@@ -22,8 +23,13 @@ const AvtalekortMobil: FunctionComponent<{
         {avtaler.map((avtale: Avtale) => {
             const ulestVarsel = varsler.find(value => value.avtaleId === avtale.id);
             return (
-                <a className={cls.element('lenke')} href={pathTilKontaktinformasjonSteg(avtale.id)}>
-                    <LenkepanelBase key={avtale.id} className={cls.className}>
+                <a className={cls.element('lenke')}>
```

Comment:
Intelliji vil ha href for <a> for tilgjenglighet aspekt og at det kan brukes fra tastatur,

Reply:
Den funker med tastatur normalt, litt usikker hvorfor Intelliji klager :)

Reply:
Tar bort `<a>`-elementet

Reply:
Lurer på om det er nødvendig å bruke <a lenger så kangt lenken har blitt inne i <LenkPanelBase ? 
Lokalt så kan det slettes ser ut sånn for meg